.github/workflows/ci-generate.yaml

```diff
@@ -0,0 +1,51 @@
+name: CI
+
+on:
+  pull_request:
+     branches:
+       - main
+
+jobs:
+  check-drift:
+    runs-on: ubuntu-latest
+    env:
+      REPO: ${{ github.event.repository.name }}
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Install Rust toolchain
+        uses: dtolnay/rust-toolchain@stable
+
+      - name: Install cargo-generate
+        uses: taiki-e/install-action@v2
+        with:
+          tool: cargo-generate
```

Comment:
Can this use `binstall` instead to save time? See `release.yaml` for an example, where we use it to install `trunk`.

Reply:
I believe it already does: 
![image](https://github.com/user-attachments/assets/6f5c855a-0d53-48f3-9876-4c87b63a8f67)

Reply:
Interesting. It seems this action has a hardcoded list of supported tools, which are mostly installed from GitHub releases, and uses `cargo-binstall` as a fallback. In either case it wouldn't have to compile `cargo-generate`, so I guess this is fine.